code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -27,6 +27,14 @@ This file's folder contains:
 	clock_unsorted.dm: Anything else with no place to be
 
 	clockcult defines are in __DEFINES/clockcult.dm
+
+Credit where due:
+1. SkowronX from /vg/ for the original idea and code
+2. FuryMcFlurry from /vg/ for many of the assets
+3. PJB3005 from /vg/ for the failed continuation PR
```

Comment:
*AHEM.*

I have simply shifted priorities to more pressing matters.

ONE DAY I WILL MAKE /vg/ CLOCKCULT.

IT'S GONNA BE GLORIOUS.

JUST YOU WATCH.